syncopy/io/load_tdt.py

```diff
@@ -0,0 +1,700 @@
+            tsq.seek(-24, os.SEEK_END)
+            header.stop_time = np.fromfile(tsq, dtype = np.float64, count = 1)
+
+        [data.info.tankpath, data.info.blockname] = os.path.split(os.path.normpath(self.block_path))
+        data.info.start_date = datetime.fromtimestamp(header.start_time[0])
+        if not np.isnan(header.start_time):
+            data.info.utc_start_time = data.info.start_date.strftime('%H:%M:%S')
+        else:
+            data.info.utc_start_time = np.nan
+
+        if not np.isnan(header.stop_time):
+            data.info.stop_date = datetime.fromtimestamp(header.stop_time[0])
+            data.info.utc_stop_time = data.info.stop_date.strftime('%H:%M:%S')
+        else:
+            data.info.stop_date = np.nan
+            data.info.utc_stop_time = np.nan
+
+        if header.stop_time > 0:
+            data.info.duration = data.info.stop_date - data.info.start_date  # datestr(s2-s1, 'HH:MM:SS')
+
+        tsq.seek(40, os.SEEK_SET)
+
+        read_size = 10000000 if self.t2 > 0 else 50000000
+        header.stores = StructDict()
+        code_ct = 0
+        while True:
+            heads = np.frombuffer(tsq.read(read_size * 4), dtype = np.uint32)
+            rem = len(heads) % 10
+            if rem != 0:
+                warnings.warn('Block did not end cleanly, removing last {0} headers'.format(rem), Warning, stacklevel = 2)
+                heads = heads[:-rem]
+
+            # reshape so each column is one header
+            heads = heads.reshape((-1, 10)).T
+
+            # check the codes first and build store maps and note arrays
+            codes = heads[2, :]
+
+            good_codes = codes > 0
+            bad_codes = np.logical_not(good_codes)
+
+            if np.sum(bad_codes) > 0:
+                warnings.warn('Bad TSQ headers were written, removing {0}, keeping {1} headers'.format(sum(bad_codes), sum(good_codes)), Warning, stacklevel = 2)
+                heads = heads[:, good_codes]
+                codes = heads[2, :]
+            # get set of codes but preserve order in the block
+            store_codes = []
+            unique_codes, unique_ind = np.unique(codes, return_index = True)
+            for counter, x in enumerate(unique_codes):
+                store_codes.append({
+                    'code': x,
+                    'type': heads[1, unique_ind[counter]],
+                    'type_str': self.code_to_type(heads[1, unique_ind[counter]]),
+                    'ucf': self.check_ucf(heads[1, unique_ind[counter]]),
+                    'epoc_type': self.epoc_to_type(heads[1, unique_ind[counter]]),
+                    'dform': heads[8, unique_ind[counter]],
+                    'size': heads[0, unique_ind[counter]],
+                    'buddy': heads[3, unique_ind[counter]],
+                    'temp': heads[:, unique_ind[counter]]
+                })
+
+            # Looking for only Mark, PDi\ and PDio
+            looking_for = ["Mark", "PDio", 'LFPs', "PDi\\"]  #
```

Comment:
Could this be good to expose to the users? If they want to look for other targets?